envi/archs/arm/disasm.py

```diff
@@ -1885,17 +1883,17 @@ def repr(self, op):
         offreg = arm_regs[self.offset_reg][0]
         shname = shift_names[self.shtype]
         if self.shval != 0:
-            shval = "%s #%d"%(shname,self.shval)
+            shval = ", %s #%d"%(shname,self.shval)
         elif self.shtype == S_RRX:
             shval = shname
         else:
-            shval = ""
+            shval = ""                                                                                           
```

Comment:
Turning on white space display so I see spare spaces...
Is the removal of the space just for looks or is there an issue with them?